docs/user-guide/diffing.md

```diff
@@ -53,6 +55,18 @@ spec:
     - .spec.template.spec.initContainers[] | select(.name == "injected-init-container")
 ```
 
+To ignore fields owned by specific managers defined in your live resources:
+```yaml
+spec:
+  ignoreDifferences:
+  - group: *
+    kind: *
+    managedFieldsManagers:
+    - kube-controller-manager
+```
+
+The above configuration will ignore differences from all fields owned by `kube-controller-manager` for all resources belonging to this application.
```

Comment:
Please add example of `managedFieldsManagers` for system-level configuration.

Also please add `managedFieldsManagers` to https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/application.yaml and https://github.com/argoproj/argo-cd/blob/master/docs/operator-manual/argocd-cm.yaml .

Comments might be minimal. These two files are used to quickly find which settings are available.

Reply:
done